src/XamlX.IL.Cecil/CecilType.cs

```diff
@@ -174,6 +174,8 @@ public IXamlType GetEnumUnderlyingType()
                 return TypeResolveContext.Resolve(Definition.GetEnumUnderlyingType());
             }
 
+            public bool IsFunctionPointer => Definition.IsFunctionPointer;
```

Comment:
It's expected to always be false here, right?

Reply:
Yes. Only Cecil's `FunctionPointerType` returns `true` here, and it's not a `TypeDefinition`.
I can change it to `false` for clarity/

Reply:
Just a comment is fine too

Reply:
Possibly a comment on CecilFunctionPointerType usage too in CecilTypeCache. I had a hard time to understand all the logic in these files with my recent PRs :). Cecil really requires a degree sometimes.